datalake/archive.py

```diff
@@ -269,16 +269,14 @@ def _s3_bucket(self):
         # bucket. And this will 403.
         return self._s3_conn.get_bucket(self._s3_bucket_name, validate=False)
 
-    _KEY_FORMAT = '{prefix}-{where}/{what}/{start}/{id}'
+    _KEY_FORMAT = '{id}'
```

Comment:
I think we should go with `{id}/data` instead of just `{id}`. This paves the path for other things in the future (e.g., abandoning s3 metadata facility).